scheduler-bindings/src/lib.rs

```diff
@@ -424,30 +432,87 @@ pub mod worker_message_types {
         // pub const COMMIT_CANCELLED: u8 = 102;
     }
 
-    /// Tag indicating [`Resolved`] inner message.
-    pub const RESOLVED: u8 = 1;
+    /// Tag indicating [`CheckResponse`] inner message.
+    pub const CHECK_RESPONSE: u8 = 1;
 
-    /// Resolving was unsuccessful.
-    pub const RESOLVE_FAILURE: u8 = 0;
-    /// Resolving was successful.
-    pub const RESOLVE_SUCCESS: u8 = 1;
+    pub mod parsing_and_sanitization_flags {
+        /// Flag set if parsing and sanitization failed.
+        pub const FAILED: u8 = 1 << 1;
+    }
+
+    pub mod status_check_flags {
+        /// Flag set if status checks were requested.
+        pub const REQUESTED: u8 = 1 << 1;
+        /// Flag set if status checks were performed. A previous failure
+        /// could have caused checks to be skipped.
+        pub const PERFORMED: u8 = 1 << 2;
+        /// Flag set if status checks failed due to the transaction being
+        /// too old.
+        pub const TOO_OLD: u8 = 1 << 3;
```

Comment:
Technically a randomly generated blockhash would not be too old but just simply "invalid" right? I assume this error is for blockhash failure on TXs that are not using durable nonce?

Reply:
ahh yep, i had assumed you had a way of splitting those out. if nonce is getting mixed in would do something like the invalid lifetime name